Library/ViewModels/ProjectPageViewModel.swift

```diff
@@ -493,7 +497,7 @@ public final class ProjectPageViewModel: ProjectPageViewModelType, ProjectPageVi
         .skipNil()
 
     // We skip the first one here because on `viewDidLoad` we are setting .overview so we don't need a useless emission here
-    self.updateDataSource = self.projectNavigationSelectorViewDidSelectProperty.signal
+    let dataSourceUpdate = self.projectNavigationSelectorViewDidSelectProperty.signal
```

Comment:
I broke out the main `updateDataSource` logic (that configures the rest of the Project Page's tableview) into it's own dataSourceUpdate property to make it easier to combine with `self.similarProjectsUseCase.similarProjects.signal`.